frontend/src/views/files/Preview.vue

```diff
@@ -339,9 +339,7 @@ export default {
     }, 500),
     close() {
       this.$store.commit("updateRequest", {});
-
-      let uri = url.removeLastDir(this.$route.path) + "/";
-      this.$router.push({ path: uri });
+      this.$router.go(-1);
```

Comment:
The `go` method cannot be used because:

- We do not have control of the user history, the previous entry is not necessarily from FileBrowser.
- The user history is empty when accessing the page directly by its URL, since there's no previous page nothing will happen.

Reply:
Thank you for your review. 
Then, what should I use? I have no idea.  Or just use "push", it has a bug here. as I tested, if I click the "back" button again, it will revisit the pic. 

Hope you can make a PR for it.  Thanks again!

Reply:
I think that the changes addressed on the methods `prev()` and `next()`, keeping `close()` untouched, are enough for this PR. If some browser is not behaving correctly with these changes, please add information how to reproduce the issue and will take a look.

Reply:
Ok, I will change it back.